.github/workflows/stockwatch-tests.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         run: poetry install --no-interaction --no-root --only testrunner
 
       - name: Run tests
-        run: poetry run tox -e style_and_typing -e py310
+        run: poetry run tox -e style -e typing -e py310
```

Comment:
this change was already done in another merge, will probably cause conflicts when merging. In principle the commit could just be removed.

Reply:
solved